lessons/advanced/strings-in-depth.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: page
+title: Strings in-depth 
+category: advanced
+order: 9
+lang: en
+---
+
+String manipulation is one of our must to have skills nowadays, ignoring the programming language. That's why we are including an in-depth on Strings.
```

Comment:
> String manipulation is one of our must ~~to~~ have skills nowadays~~, ignoring the programming language~~. ~~That's why we are including an in-depth on Strings.~~ 

Most of the above doesn't fit into the typical overview of Elixir School. Consider finishing it off with something about codepoints and graphemes.